plugins/announcements/src/components/AnnouncementPage/AnnouncementPage.tsx

```diff
@@ -41,9 +41,9 @@ const AnnouncementDetails = ({
   const subHeader = (
     <span>
       By{' '}
-      <EntityPeekAheadPopover entityRef={announcement.publisher}>
```

Comment:
What was the reason behind replacing the `EntityPeekAheadPopover` component with a `Link`?

Reply:
Good catch, I was too fast on copy/past. It's fixed